common/tests/test_metrics.py

```
@@ -0,0 +1,309 @@
+from unittest.mock import MagicMock, patch
+
+from django.test import TestCase
+
+from common.metrics import metrics
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _counter_value(counter, *label_values):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_counter_value`

(ANN202)